sdks/wasm/sdk/Mono.WebAssembly.Build/Mono.WebAssembly.Build.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <Description>Mono.WebAssembly build targets. This package is not intended to be referenced directly.</Description>
+    <BuildOutputTargetFolder>build\netstandard2.0</BuildOutputTargetFolder>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" ExcludeAssets="runtime" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" ExcludeAssets="runtime" PrivateAssets="all" />
+    <PackageReference Include="System.CodeDom" PrivateAssets="all" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="..\..\..\out\wasm-bcl\wasm_tools\monolinker.exe" Link="monolinker.exe" CopyToOutputDirectory="PreserveNewest" Pack="true" PackagePath="$(BuildOutputTargetFolder)" />

```

Comment:
Is PreserveNewest what we want here? I think we want CopyAlways so that you get the correct, consistent set of outputs if you have a latest-master checkout and you revert to an older rev, etc

Reply:
Good point, will fix